packages/lib/services/plugins/utils/manifestFromObject.ts

```diff
@@ -36,6 +36,13 @@ export default function manifestFromObject(o: any): PluginManifest {
 		return o.screenshots;
 	};
 
+	const getIcons = (): Icons => {
+		for (const size of [16, 32, 48, 128]) {
+			if (o.icons[size as keyof Icons]) return o.icons;
```

Comment:
This code's not good - if o.icons is undefined it throws an error. Please fix

Reply:
You had the code right in the previous commit, why did you change it? When addressing review comments, please limit the changes to just what was asked, otherwise it never ends

Reply:
I fixed it now

Reply:
Ok. Thanks.